src/components/landing/header.tsx

```diff
@@ -86,7 +84,11 @@ function MobileNavigation() {
             <MobileNavLink href="#features">Features</MobileNavLink>
 
             <hr className="m-2 border-slate-300/40" />
-            <MobileNavLink href="/auth">Sign in</MobileNavLink>
+            <MobileNavLink href="/">Github</MobileNavLink>
+            <hr className="m-2 border-slate-300/40" />
+            <MobileNavLink href="/">Twitter</MobileNavLink>
+            <hr className="m-2 border-slate-300/40" />
+            <MobileNavLink href="/">Discord</MobileNavLink>
```

Comment:
It will be great to populate this with the actual links; otherwise, users will always be directed to the landing page.
Since there is no Twitter as well, we can just get rid of that and maintain just Discord and GitHub.

Github: https://AmoabaKelvin/ishortn.ink
Discord: https://discord.com/invite/S66ZvMzkU4